code/modules/jobs/job_types/curator.dm

```diff
@@ -19,6 +19,7 @@
 
 	display_order = JOB_DISPLAY_ORDER_CURATOR
 	departments = DEPARTMENT_SERVICE
+	rpg_title = "Veteran Adventurer"
```

Comment:
How about Talesmaster instead of Veteran Adventurer? because they look advanced than Shaft Miner.

Reply:
I like the original more.